int32 id
---
bool setclear_error_accepted